src/dispatch/event/service.py

```diff
@@ -113,14 +113,17 @@ def log(
     description: str,
     incident_id: int,
     individual_id: int = None,
-    started_at: datetime = datetime.datetime.now(),
+    started_at: datetime = None,
```

Comment:
Can't this be `started_at: Optional[datetime] = datetime.datetime.now()`?

Reply:
Would that behave differently from what I had before? I decided to move it to the body of the function, because I noticed that the value was the same for all events in the database. Moving it there solved the problem.